sysutils/snapraid/Portfile

```
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        amadvance snapraid 12.3 v
+
+name                snapraid
+version             12.3
+categories          sysutils
+
+license             GPL-3
+maintainers         {anquieta @anquieta}
+description         Backup program for disk arrays
+long_description    SnapRAID stores parity information of your data and \
+                    it recovers from up to six disk failures
+homepage            http://www.snapraid.it/
+
+github.tarball_from downloads
+
+distname            ${github.project}-${github.version}
+
+master_sites        ${github.homepage}/releases/download/${github.tag_prefix}${github.version}/
+
+checksums           rmd160  0a8bb0ef5a7b1c925138b683ba811b7913d5d612 \
+                    sha256  1f27aba6e5db3929474da9c472fc4a3bc51d4a83aa5385af6b09afc9f68d6d44 \
+                    size    746308
+
+platforms           any
+
+use_parallel_build  yes
+
+depends_build       port:autoconf \
```

The best practice is to have `depends_build-append` because often certain portgroups may already add things to the various `depends_`:

```suggestion
depends_build-append \
                    port:autoconf \
```